Demo/Demo/ViewModels/PayPalViewModel.swift

```diff
@@ -0,0 +1,135 @@
+import UIKit
+import PayPalNativeCheckout
+import PayPalCheckout
+import PaymentsCore
+
+class PayPalViewModel: ObservableObject, PayPalDelegate {
```

Comment:
Good idea to separate this PayPalViewModel out of the BaseViewModel!

Reply:
However, I think `BaseViewModel` was made with the intent of being subclassed by each feature's ViewModel. It was meant to house shared functionality b/w all ViewModels. So for example, we have some redundancies now b/w the BaseVM and this new VM such as creating an access token and fetching an orderID.

Reply:
The idea for this PR was to show how could the possible architectures be for any feautre we have to demo. So i think as a first step in that direction, i considered NXO only, and if we agree with the architecture (SwiftUI, viewmodel, usecases, etc), then we can work on the other features and try to create Base classes if we see more redundancies.
But also, thats what the useCases are for, viewmodel can use the same useCases.